client/src/main/java/org/evosuite/coverage/FitnessFunctions.java

```diff
@@ -216,6 +256,27 @@ public static TestSuiteFitnessFunction getFitnessFunction(Criterion criterion) {
 			return new OutputCoverageFactory();
 		case INPUT:
 			return new InputCoverageFactory();
+		//modified start harsh
+		case VDDU:
+		case VMDDU:
+		case VCDDU:
+//		case VCMDDU1:
+//		case VCMDDU2:
```

Comment:
If these two are commented our in here, should they be remove from the Properties class?